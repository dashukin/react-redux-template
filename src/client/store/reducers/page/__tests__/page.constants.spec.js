import pageConstants from '../page.constants';

describe('page.constants', () => {
  it('should match snapshot', () => {
    expect(pageConstants).toMatchSnapshot();
  });
});
