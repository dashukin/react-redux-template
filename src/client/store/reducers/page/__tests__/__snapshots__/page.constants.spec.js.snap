// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page.constants should match snapshot 1`] = `
Object {
  "@@redux-first-router/NOT_FOUND": "NotFoundPage",
  "HOME": "HomePage",
}
`;
